spec/npm/asciidoctor-all.spec.js

```diff
@@ -21,5 +37,76 @@ describe('Node.js', function () {
       expect(doc.attributes.$fetch('docname')).toBe('test.adoc');
     });
   });
+
+  describe('Converting file', function() {
+    it('should be able to convert a file', function() {
+      var expectFilePath = __dirname + '/test.adoc.html';
+      removeFile(expectFilePath);
+      try {
+        var doc = Asciidoctor.$convert_file(__dirname + '/test.adoc', null);
```

Comment:
@mojavelinux Even without `to_file` the returned value is a `Document` and a file is written `test.adoc.html`. 
https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor.rb#L1575-L1576

In my case I think `to_file` is `nil`: https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor.rb#L1424
Is that the correct behavior ?